health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/FormulaConfiguration.js

```
@@ -89,18 +198,120 @@ const FormulaConfiguration = ({onSelect,  category,  customProps})=>{
+                                      sortFooterChildren={true}
+                                      onClose={() => {
+                                        setShowPopUp(false);
+                                      }}
+                          ></PopUp> } 
+
+                        {formulasPopUP && <PopUp
+                                    className={"popUpClass"}
+                                    type={"default"}
+                                    heading={t("CONFIRM_NEW_FORMULA")}
+                                    equalWidthButtons={true}
+                                      children={[
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)